cmd/protoc-gen-crd/openapiGenerator.go

```diff
@@ -304,7 +304,23 @@ func (g *openapiGenerator) generateFile(
 	for name, cfg := range genTags {
 		log.Println("Generating", name)
 		group := cfg["groupName"]
-		version := cfg["version"]
+
+		versionsString := cfg["versions"]
+		versions := strings.Split(versionsString, ",")
+		var storageVersion string
+		if version := cfg["version"]; version != "" {
+			if len(versions) == 0 {
+				log.Fatal("can only set versions or version")
+			}
+			if _, f := cfg["storageVersion"]; f {
+				// Old way: single version specifies explicitly
+				storageVersion = version
+			}
+			versions = []string{version}
+		} else {
+			// New way: first one is the storage version
+			storageVersion = versions[0]
```

Comment:
In Istio's history, have we ever changed the storageVersion for any of the CRDs? I am curious if a process for that exists.

Reply:
I think we change networking to beta I think. I think we just kind of... did it. I don't recall discussing it